packages/core/src/animation/AnimatorControllerParameter.ts

```diff
@@ -0,0 +1,11 @@
+export type AnimatorControllerParameterValue = number | string | boolean;
```

Comment:
Don't need

Reply:
I think the types of values need to be limited